Libraries/Components/Clipboard/Clipboard.android.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule Clipboard
+ */
+'use strict';
+
+var Clipboard = require('NativeModules').Clipboard;
+
+module.exports = {
+  getString(cb,errorCb){
+    if(!errorCb){
+      errorCb = ()=>{};
+    }
+    Clipboard.getString(cb,errorCb);
```

Comment:
space after , before errorCb

Also, can we call these "success, error"? I don't think we use the "cb" abbreviation elsewhere.